.env.example

```diff
@@ -2,6 +2,7 @@
 MODE=dev
 
 # URLs
+# You can add multiple WEB3 nodes, separated by a semilcolon ';'. e.g. WEB3=http://localhost:8545;http://localhost:8550
 WEB3=http://localhost:8545
```

Comment:
I guess this should be remove as it duplicates line above.

Reply:
The line above is just a comment.
I would leave this line so that people can c/p this .env.example, rename to .env and get going.

Reply:
Ok, GH wrapped the line after e.g. so it looked for me like

```
WEB3=http://localhost:8545;http://localhost:8550
WEB3=http://localhost:8545
```